pageserver/src/page_service.rs

```diff
@@ -251,15 +459,17 @@ struct Connection {
     buffer: BytesMut,
     init_done: bool,
     conf: PageServerConf,
+    runtime: Arc<Runtime>,
 }
 
 impl Connection {
-    pub fn new(conf: PageServerConf, socket: TcpStream) -> Connection {
+    pub fn new(conf: PageServerConf, socket: TcpStream, runtime: &Arc<Runtime>) -> Connection {
         Connection {
             stream: BufWriter::new(socket),
             buffer: BytesMut::with_capacity(10 * 1024),
             init_done: false,
             conf,
+            runtime: runtime.clone(),
```

Comment:
When cloning an `Arc` it's preferable to write `Arc::clone(runtime)` because if the inner type also implements `clone`, it's not clear which one you meant.